examples/quickstart/jupyter-notebooks/druidapi/README.md

```diff
@@ -36,21 +39,25 @@ At present, the best way to use `druidapi` is to clone the Druid repo itself:
 git clone git@github.com:apache/druid.git
 ```
 
-`druidapi` is located in `examples/quickstart/jupyter-notebooks/druidapi/`
+`druidapi` is located in `examples/quickstart/jupyter-notebooks/druidapi/`.
+From this directory, install the package and its dependencies with pip using the following command:
 
-Eventually we would like to create a Python package that can be installed with `pip`. Contributions
-in that area are welcome.
+```
+pip install .
+```
```

Comment:
Same note as above.

Reply:
This is referenced in the previous few lines:
> `druidapi` is located in `examples/quickstart/jupyter-notebooks/druidapi/`.
From this directory, install the package ...